docs/guides/universal-profile/check-if-address-is-universal-profile.md

```diff
@@ -24,95 +59,224 @@ if (!isAddress(address)) {
 
 ```javascript
-// This ABI can also be imported from @lukso/lsp-smart-contracts npm package
-
-const getDataABI = [
-  {
-    inputs: [
-      {
-        internalType: 'bytes32',
-        name: 'dataKey',
-        type: 'bytes32',
-      },
-    ],
-    name: 'getData',
-    outputs: [
-      {
-        internalType: 'bytes',
-        name: 'dataValue',
-        type: 'bytes',
-      },
-    ],
-    stateMutability: 'view',
-    type: 'function',
-  },
-];
+import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import { SupportedStandards } from '@lukso/lsp-smart-contracts/constants';
+import Web3 from 'web3';
+
+const web3 = new Web3('https://rpc.l16.lukso.network');
+
+const universalProfileAddress = "0x..."; // The address of the contract that you are verifying
+const unviersalProfile = new web3.eth.Contract(UniversalProfile.abi, universalProfileAddress);
+
+const supportedStandard = await unviersalProfile.methods['getData(bytes32)'](SupportedStandards.LSP3UniversalProfile.key).call();
+
+if (supportedStandard !== SupportedStandards.LSP3UniversalProfile.value) {
+  throw new Error('Address does not support LSP3UniversalProfile standard');
+}
 ```
 
-</details>
+<!-- prettier-ignore-end -->
+
+  </TabItem>
+
+  <TabItem value="ethersjs" label="ethers.js">
 
 <!-- prettier-ignore-start -->
 
 ```javascript
-import {INTERFACE_IDS, SupportedStandards} from '@lukso/lsp-smart-contracts/constants';
+import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import { SupportedStandards } from '@lukso/lsp-smart-contracts/constants';
+import { ethers } from 'ethers';
+
+const provider = new ethers.JsonRpcProvider('https://rpc.l14.lukso.network');
 
-const contract = new web3.Contract(getDataABI, address);
+const universalProfileAddress = '0x...'; // The address of the contract that you are verifying
+const unviersalProfile = new ethers.Contract(universalProfileAddress, UniversalProfile.abi, provider);
 
-const supportedStandard = await contract.methods['getData(bytes32)'](SupportedStandards.LSP3UniversalProfile.key).call();
+const supportedStandard = await unviersalProfile['getData(bytes32)'](SupportedStandards.LSP3UniversalProfile.key);
 
 if (supportedStandard !== SupportedStandards.LSP3UniversalProfile.value) {
   throw new Error('Address does not support LSP3UniversalProfile standard');
 }
 ```
 
 <!-- prettier-ignore-end -->
+
+  </TabItem>
+
+</Tabs>
+
+## Final Code
+
+<Tabs>
+  
+  <TabItem value="web3js" label="web3.js">
+
+<!-- prettier-ignore-start -->
+
+```javascript
+import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
+import { INTERFACE_IDS, SupportedStandards } from '@lukso/lsp-smart-contracts/constants';
+import Web3 from 'web3';
+
+const web3 = new Web3('https://rpc.l16.lukso.network');
+
+if (!web3.utils.isAddress(address)) {
+  throw new Error('Invalid address');
+}
+
+// We assume that the contract is a Universal Profile
```

Comment:
this comment seems a bit weird to me, we are checking if it is a UP so we arent assuming its a UP. ISnt that the point of the guide?

Reply:
At the point where the comment is made the type of the contract is still unknown as the checks weren't made. We first assume and then verify. That was the thought process.